docker/android-sdk/tools/androidctl/sdk.py

```diff
@@ -73,10 +73,11 @@ def uninstall(path=sdk_path):
 def getProxySettings(proxy=https_proxy):
   proxy_settings = None
   if proxy:
-    parts = proxy.split(':')
-    proxy_host = parts[0]
-    proxy_port = None
-    if parts[1]:
-      proxy_port = parts[1]
+    if '://' not in proxy:
```

Comment:
correct/safe way is to use "not" before the expression: `if not '://' in proxy:`